examples/java/src/main/java/org/apache/beam/examples/snippets/Snippets.java

```diff
@@ -15,7 +15,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.beam.examples;
+package org.apache.beam.examples.snippets;
```

Comment:
@melap Will moving this break something in the website?

Reply:
This should be fine -- the path to the file and the snippet tags are the two things the website uses.

Reply:
@davidcavazos in case he sees anything on his side that this might affect